utils/queryApiGateway.ts

```diff
@@ -112,6 +113,51 @@ export function getUniqueNumber(req: IncomingMessage): string {
   return req.headers[idHeaderName.toLowerCase()] as string
 }
 
+/**
+ * Check if the API responded with a null response body
+ *
+ * We do not expect any of these in production, in theory.
+ */
+export function reponseIsNull(apiBody: Claim): boolean {
+  const response: Claim = JSON.parse(JSON.stringify(apiBody)) as Claim
+  delete response.uniqueNumber
+
+  // This is the response for a bad UniqueNumber, or a UniqueNumber without a claim
+  const nullResponseShort = {
+    claimDetails: null,
+    hasCertificationWeeksAvailable: false,
+    hasPendingWeeks: false,
+    pendingDetermination: null,
+  }
+
+  // This is an unexpected null response we are seeing in production
+  // catch here to avoid showing any bad data
+  const nullResponseLong = {
+    claimDetails: {
+      programType: '',
+      benefitYearStartDate: null,
+      benefitYearEndDate: null,
+      claimBalance: null,
+      weeklyBenefitAmount: null,
+      lastPaymentIssued: null,
+      lastPaymentAmount: null,
+      monetaryStatus: '',
+    },
+    hasCertificationWeeksAvailable: false,
+    hasPendingWeeks: false,
+    pendingDetermination: [],
+  }
```

Comment:
I feel like hard coding these "null" responses is not the right way to approach this, but I think that #150 is actually the right way, so this seems like a good enough approach until we get #150 done.